src/main/java/com/nvidia/spark/rapids/jni/CastStrings.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.jni;
+
+import ai.rapids.cudf.*;
+
+/** Utility class for casting between string columns and native type columns */
+public class CastStrings {
+  static {
+    NativeDepsLoader.loadNativeDeps();
+  }
+
+  public static ColumnVector stringToInteger(ColumnView cv, boolean ansiMode, DType type) {
```

Comment:
Nit: `strings` seems a bit redundant here since it's a static method of the `CastStrings` class, so it's already known  we're casting from a string.
```suggestion
  public static ColumnVector toInteger(ColumnView cv, boolean ansiMode, DType type) {
```